src/components/TagSet/TagSetFilter.tsx

```diff
@@ -103,8 +99,31 @@ export const TagSetFilter: React.FC<TagSetFilterProps> = chakra(
                     color={dismissButtonColor}
                     width="12px"
                   />
-                ) : null}
-              </Button>
+                </Button>
+              ) : (
+                <Box
+                  data-testid="filter-tags"
+                  id={`ts-filter-${tagSet.id}-${key}`}
+                  sx={styles.base}
+                >
+                  {tagSet.iconName ? (
+                    <Icon
+                      align="left"
+                      color={iconColor}
+                      data-testid="ts-icon"
+                      name={tagSet.iconName}
+                      size="small"
+                    />
+                  ) : null}
+                  <span
+                    style={{
+                      fontWeight: "var(--nypl-fontWeights-button-default)",
```

Comment:
Does moving this rule to the `Box` also work? Just to reduce having another span.